src/components/Notifications/index.tsx

```diff
@@ -70,19 +70,13 @@ const getTransformedOptions = (
 	}
 };
 
-type NotificationContainerProps = {
-	children: React.ReactNode;
-	type?: NotificationOptions['type'];
-	onDismiss: NotificationOptions['onDismiss'];
-	id: NotificationOptions['id'];
-};
-
 const NotificationContainer = ({
-	children,
+	content,
 	type,
 	onDismiss,
 	id,
-}: NotificationContainerProps) => {
+	...props
+}: NotificationOptions) => {
 	return (
 		<FullWidthContainer
```

Comment:
BTW, any reason we didn't inline the `width: 100%;` here and instead we preferred the `styled(Alert)` way?
Imo it's more readable when everything is in one place, we can avoid some precedence edge cases, and such styled system props should perform similarly.

Reply:
No clue what the reasoning was behind it, this was before my time here 😅